index.html

```diff
@@ -45,6 +45,106 @@ <h1>
 
         <a href="#" class="arrow"><img src="images/Disabled.png" alt="" /></a>
       </section>
+
+      <section class="my-works">
+        <h2>My Recent Works</h2>
+        <img src="images/Indicator.png" alt="indicator" />
+        <div class="Works">
+          <div>
+            <div class="image-box"></div>
+            <div class="text-box">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+              <div class="tech">
+                <span>Ruby on rails</span>
+                <span>css</span>
+                <span>html</span>
+                <span>Javascript</span>
+              </div>
+              <div class="see-project"><a href="#">See project</a></div>
```

Comment:
- [ ] The button (e.g., "See project") is a link `a` or` button` tag, and follow the button styling: Active, Hover and Pressed. Kindly add the Active, Hover & pressed styling on all the buttons.
![gedgsdg](https://user-images.githubusercontent.com/26767383/153494287-5e639a2b-fff4-4f26-8751-8f6b29a5d453.PNG)